awx/ui/src/screens/Inventory/shared/Inventory.helptext.js

```diff
@@ -188,6 +188,40 @@ const getInventoryHelpTextStrings = () => ({
       </>
     );
   },
+  constructedInventorySourceVars: () => {
+    const docsUrl =
+      'https://docs.ansible.com/ansible/latest/collections/ansible/builtin/constructed_inventory.html';
```

Comment:
Use the same utility here https://github.com/ansible/awx/blob/devel/awx/ui/src/util/getDocsBaseUrl.js